packages/connect/src/api/cardano/cardanoOutputs.ts

```diff
@@ -43,3 +57,40 @@ export const transformOutput = (output: any): OutputWithTokens => {
 
     return result;
 };
+
+export const sendOutput = async (typedCall: any, outputWithData: OutputWithData) => {
+    const MAX_CHUNK_SIZE = 512;
```

Comment:
I don't like this number being different than the number in FW (even though it represents the same number of bytes). I would prefer converting the data from hex-encoded string to bytes asap and dealing only with bytes later on - is that possible?

Reply:
I wouldn't convert to bytes before sending, see [comment](https://github.com/vacuumlabs/trezor-suite/pull/1#discussion_r907136861) above.

Perhaps this would be better?
```suggestion
    const MAX_CHUNK_SIZE = 256 * 2; // 256 hex encoded bytes
```

Reply:
I added my suggestion in https://github.com/vacuumlabs/trezor-suite/pull/1/commits/58062096a0b3ab7b5e023ad1e11789368ceba375.

Reply:
Alright, your suggestion seems reasonable to me 👍